docs/content/about-iota/tokenomics/gas-in-iota.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Gas in IOTA
+title: Transaction Pricing
```

Comment:
I think there may have been a git mixup here, these changes don't seem to match the PR's description